Web/Admin.UI/src/app/components/tenant/acquisition-log/acquisition-log-view/acquisition-log-view.component.scss

```
@@ -121,6 +121,40 @@ $skipped-status: #ffbf00;
+  border-radius: 6px;
+  padding: 4px 10px;
+  background-color: #f9fafb;
+}
+
+.cancel-age-field {
+  width: 110px;
+  flex: 0 0 auto;
+
+  input { text-align: right; font-weight: 600; }
+
+  ::ng-deep .mat-mdc-form-field-subscript-wrapper { display: none; }
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)